common/params.py

```diff
@@ -407,10 +409,10 @@ def f(key, val):
       pp = params.get(k)
       if pp is None:
         print("%s is None" % k)
-      elif all(ord(c) < 128 and ord(c) >= 32 for c in pp):
+      elif all(c < 128 and c >= 32 for c in pp):
```

Comment:
could use `chr(c) in string.printable`

Reply:
`string.printable` returns some (white) ASCII characters with value under 32